README.md

```diff
@@ -1,5 +1,9 @@
 All times are EST. Let the horror begin.
 
+##### 14 April 2020 7:30 PM
+
+    <slembcke> massages cheddar into @mysteriouspants 's hair
```

Comment:
🧀

Reply:
What's with the extra space?!

Reply:
I think it's an artifact of copy-pasting from Discord. Good catch.